engine/runtime-parser/src/main/java/org/enso/compiler/core/IR.java

```diff
@@ -54,7 +54,12 @@ public interface IR {
   MetadataStorage passData();
 
   /** The source location that the node corresponds to. */
-  Option<IdentifiedLocation> location();
+  IdentifiedLocation identifiedLocation();
```

Comment:
This is the change

Reply:
minor: Please mention that the return value can be `null`. I am already confused in our codebase about stuff that can be null and that cannot be null.